include/strong_type/strong_type.hpp

```diff
@@ -16,29 +16,65 @@ namespace strong
 
 namespace impl
 {
+  template<class...>
+  struct has_tag : std::false_type {};
+
+  template<class Tag, class SingleTag>
+  struct has_tag<Tag, SingleTag>
+  {
+    static constexpr bool value{std::is_same<Tag, SingleTag>::value};
+  };
+
+  template<class Tag, class FirstTag, class... Tags>
+  struct has_tag<Tag, FirstTag, Tags...>
+  {
+    static constexpr bool value{std::is_same<Tag, FirstTag>::value
+                                || has_tag<Tag, Tags...>::value};
+  };
+
```

Comment:
I believe this is better done using `std::disjunction<>`.

```Cpp
namespace impl
{
  template <typename T, typename ... Tags>
  struct has_tag {
    static constexpr bool value{std::disjunction_v<std::is_same<T, Tags>...>};
  };
}

Reply:
`std::disjunction<>` was introduced in C++17. Now the strong_type library can be compiled with C++14 and I think it is not worth it to break C++14 compatibility.

Reply:
Right. Thanks.